otherlibs/systhreads/st_stubs.c

```diff
@@ -259,7 +260,7 @@ static void caml_thread_remove_info(caml_thread_t th)
   th->next->prev = th->prev;
   th->prev->next = th->next;
   caml_free_stack(th->current_stack);
-  caml_delete_root(th->backtrace_last_exn);
+  caml_remove_global_root(&th->backtrace_last_exn);
```

Comment:
Are we always sure this address is going to be the one we want, rather than the copy by value for the arg?